Services/User/classes/class.ilUserProfile.php

```diff
@@ -47,7 +47,7 @@ class ilUserProfile
         "username" => array(
                         "input" => "login",
                         "maxlength" => 64,
```

Comment:
@smeyer-ilias
Sorry, I found a missing needed change, the max length need also be increased
Could you add this too?